modules/ossm-install-kiali-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-install-kiali-operator_{context}"]
+= Installing the {KialiProduct}
+:context: ossm-install-kiali-operator
+
+// TP 1 Content
+// Possible file name may change
+// Possible assembly file may change
+// Assemblies, topic map info needs to be worked out still for 3.0.
+// Kiali install content for TP1 is the same as install content for 2.x.
+
+The following steps show how to install the {KialiProduct}.
+
+[WARNING]
+====
+Do not install Community versions of the Operators. Community Operators are not supported.
+====
+
+.Prerequisites
+
+* Access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *OperatorHub*.
+
+. Type *Kiali* into the filter box to find the {KialiProduct}.
+
+. Click the *{KialiProduct}* to display information about the Operator.
```

Comment:
Here's an exception to my earlier comment about making all mentions of Kiali Operator consistency either have or not have "the" proceeding the name - even if you choose to have all instances be preceded by "the", in this case you want to remove "the" since you're technically instructing users to click on a UI element that happens to be the Operator name:
```suggestion
. Click *{KialiProduct}* to display information about the Operator.
```